File: How Many Numbers Are Smaller Than the Current Number.cpp
class Solution {
public:
    vector<int> smallerNumbersThanCurrent(vector<int>& nums) {
        vector<int> a;
        
        for (int i=0;i<=nums.size()-1 ;i++)
        {int count =0;
          for (int j=0;j<=nums.size()-1;j++)
          {if (nums[i]!=nums[j]&&nums[i]>nums[j])
             {
              count++;
              }
          }
           a.push_back(count);
        }
        return a;
           
      
  }};

File: how-many-numbers-are-smaller-than-the-current-number.py
class Solution:
    def smallerNumbersThanCurrent(self, nums: List[int]) -> List[int]:
        count=[]
        for i in  range (len(nums)):
            x=0
            for j in range(len(nums)):
                if nums[i]>nums[j]:
                 x=x+1
            count.append(x)
        return count